cardano_node_tests/utils/tx_view.py

```diff
@@ -174,10 +174,13 @@ def check_tx_view(  # noqa: C901
     if tx_raw_len_certs != loaded_len_certs:
         raise AssertionError(f"certificates: {tx_raw_len_certs} != {loaded_len_certs}")
 
-    # load transaction era
+    # load and check transaction era
     loaded_tx_era: str = tx_loaded["era"]
     loaded_tx_version = getattr(VERSIONS, loaded_tx_era.upper())
 
+    if loaded_tx_era != (tx_raw_output.era or VERSIONS.transaction_era_name).upper():
+        raise AssertionError("transaction era is not the expected")
```

Comment:
I think it's better to compare `loaded_tx_version` with `VERSIONS.DEFAULT_TX_ERA` instead of `loaded_tx_era` with `VERSIONS.transaction_era_name`. It is immediately clear that the era should be either the one in `tx_raw_output.era`, or default Tx era.

I don't get the `.upper()` part. The `loaded_tx_era` is e.g. "Alonzo", and `tx_raw_output.era.upper()` would be "ALONZO", no?

Also please include the expected and actual value in the error message.